shepherd/api/app.py

```diff
@@ -54,15 +52,15 @@ def create_app(debug=None) -> web.Application:
 
     app = web.Application(debug=debug if debug is not None else os.getenv('DEBUG', False), client_max_size=10*1024**3)
 
-    oapi.error_middleware.add_handler(NameConflictError, partial(error_handler, 409))
```

Comment:
end of our middleware?

Reply:
Not so much, but it's hidden under a common interface now.